frontend/src/types.ts

```diff
@@ -192,4 +195,28 @@ export type ValidReportDeleteFormData = {
     };
 };
 
+export type LeagueParams = {
+    league: League;
+    tier: LeagueTier | null;
+    year: number;
+};
+
+export type LeagueStats = Record<number, LeaguePlayerStats>;
+
+export type LeaguePlayerStats = {
+    name: string;
+    summary: {
+        wins: number;
```

Comment:
I believe I changed `wins` to `totalWins` and `gameCount` to `totalGames`. Should check my API PR for the proper structure.

Reply:
Checked and updated to match!